packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -1352,7 +1364,19 @@ export class FieldEditor {
                   ></fw-input>
                 </div>
               </span>
-              {toggle}
+              {dataItem?.choice_options?.resolution_timer !== undefined &&
+                renderToggle(
+                  dataItem.id,
+                  'resolution_timer',
+                  dataItem.choice_options.resolution_timer
+                )}
+              {dataItem?.choice_options?.pause_resolution_sla_timer !==
```

Comment:
@Elayaraman Shouldn't we be checking if isSlaEnabled is true here?

Reply:
Yeah I should be 
I have made the changes